ratik/README.md

```diff
@@ -18,19 +18,30 @@ org.opencadc.ratik.inventory.username={dbuser}
 org.opencadc.ratik.inventory.password={dbpassword}
 org.opencadc.ratik.inventory.url=jdbc:postgresql://{server}/{database}
 
+# local site type, true = global site, false = storage site
+org.opencadc.ratik.trackSiteLocations={true|false}
+
 # remote inventory query service (luskan)
 org.opencadc.ratik.queryService={resourceID of remote TAP service with inventory data model}
 
 # artifact uri bucket filter
 org.opencadc.ratik.buckets={uriBucket prefix or range of prefixes}
 
 # selectivity
-org.opencadc.inventory.util.ArtifactSelector={fully qualified classname of ArtifactSelector implementation}
+org.opencadc.ratik.artifactSelector={all|filter}
 
```

Comment:
The org.opencadc.ratik.artifactSelector property name in the example below for artifact-filter.sql needs to be updated.